packages/wallet/src/redux/protocols/indirect-defunding/actions.ts

```diff
@@ -1,18 +1,45 @@
-import { CommitmentReceived, WalletAction } from '../../actions';
+import { WalletAction } from '../../actions';
+import { ConsensusUpdateAction, isConsensusUpdateAction } from '../consensus-update';
+import { AdvanceChannelAction, isAdvanceChannelAction } from '../advance-channel';
+import {
+  routerFactory,
+  EmbeddedProtocol,
+  BaseProcessAction,
+  ProtocolLocator,
+} from '../../../communication';
+import { ActionConstructor } from '../../utils';
 // -------
 // Actions
 // -------
+export interface ClearedToSend extends BaseProcessAction {
```

Comment:
Is this more like `ClearedToProceed`? Is this idea that
- the protocols should be started up in order to receive messages;
- but the user might not have ok'd the defunding yet
- and once the user clears it, all sub-protocols are then cleared to send?

If that's the case, I think `ClearedToProceed` or `ClearedToDefund` might be a tad clearer, since the de-funding protocol itself doesn't actually send anything.

Reply:
renamed to `clearedToProceed`